packages/kit/src/runtime/client/client.js

```diff
@@ -1207,6 +1184,8 @@ export function create_client({ target, base, trailing_slash }) {
 				const post_update = pre_update();
 				root.$set(props);
 				post_update();
+
+				tick().then(reset_focus);
```

Comment:
I think we should only do this when the type is success. When input is invalid, it might be nice to keep focus on the currently selected thing instead.

Reply:
The currently selected thing will be the button that was used to submit the form — by keeping the focus there the only thing we're really doing is making it more likely that the user will resubmit bad data, I think

Reply:
Just tried this out - if you press enter on an input (which isn't that uncommon I'd say) the focus stays on that input after submitting it. I would be in favor of keeping the focus on that input on an invalid response.